### PR TITLE
[REF] pos*: introduce bootstrap

### PR DESCRIPTION
**Pull request by vlst-odoo**
[REF] pos*: introduce bootstrap
pos*: point_of_sale,pos_discount,pos_hr,pos_loyalty,
pos_restaurant,pos_sale,pos_sale_product_configurator,pos_six

Prior to this commit, the point_of_sale module relied on custom CSS.

To enhance user experience, increase flexibility, and simplify maintenance,
most of the custom css was replaced in this PR with Bootstrap utility classes.

Task: 3354582

Authored-by:  Xavier Luyckx (xlu) <xlu@odoo.com>
Co-authored-by:  vlst <vlst@odoo.com>,  Pedram (pebr)

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.xml [hunk 1]
```diff
@@ -45,30 +46,30 @@
                                 selectLine.bind="selectPaymentLine"
                                 updateSelectedPaymentline.bind="updateSelectedPaymentline"/>
                         </div>
-                        <div t-if="!ui.isSmall" class="button next validation" t-att-class="{ highlight: currentOrder.is_paid() and currentOrder._isValidEmptyOrder() }"
+                        <div t-if="!ui.isSmall" class="button next validation btn btn-primary btn-lg py-5 rounded-0 d-flex flex-column align-items-center justify-content-center fw-bolder" t-att-class="currentOrder.is_paid() and currentOrder._isValidEmptyOrder() ? 'highlight' : 'disabled'"
```
Comment (vlst-odoo) [inline addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.xml:49]: ?

**Review by vlst-odoo — COMMENTED** [review]
Preliminary comments

File: addons/pos_restaurant/static/src/app/floor_screen/table.xml [hunk 2]
```diff
@@ -2,15 +2,15 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="pos_restaurant.Table" owl="1">
-        <div class="table" t-on-click.stop="(ev) => props.onClick(props.table, ev)" t-att-style="style">
-            <span class="table-cover" t-att-class="{ full: fill >= 1 }" t-att-style="`height: ${Math.ceil(fill * 100)}%`"></span>
-            <span t-att-class="orderCountClass" t-att-hidden="orderCount === 0">
+        <div class="table d-flex flex-column align-items-center justify-content-between" t-on-click.stop="(ev) => props.onClick(props.table, ev)" t-att-style="style">
+            <span class="table-cover position-absolute d-block bottom-0 start-0 end-0 bg-dark bg-opacity-25 z-index-0" t-att-class="{ full: fill >= 1 }" t-att-style="`height: ${Math.ceil(fill * 100)}%`"></span>
+            <span t-att-class="orderCountClass" t-attf-class="{{ orderCount === 0 ? 'opacity-0' : ''}}">
```
Comment (vlst-odoo) [inline addons/pos_restaurant/static/src/app/floor_screen/table.xml:7]: why is this change any better?

File: addons/point_of_sale/static/src/app/pos_app.xml [hunk 3]
```diff
@@ -2,12 +2,12 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="point_of_sale.Chrome" owl="1">
-        <div class="pos" t-att-class="{ 'big-scrollbars': pos.hasBigScrollBars }">
```
[the remark this answers is not in the document]
Comment (caburj) [reply addons/point_of_sale/static/src/app/pos_app.xml:5]: Try to keep 'big-scrollbars'.

File: addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.xml (hunk 1 above)
Comment (caburj) [reply addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.xml:49]: looks like it.

**Comment by vlst-odoo** [comment]
editing restaurant table doesn't work any more

File: addons/pos_restaurant/static/src/app/floor_screen/editable_table.js [hunk 4]
```diff
@@ -147,7 +147,7 @@ export class EditableTable extends Component {
                 width: `${widthTable}px`,
```
Comment (vlst-odoo) [inline addons/pos_restaurant/static/src/app/floor_screen/editable_table.js:147]: editing tables doesn't work now

**Pull request closed without merging**